packages/test/test-version-utils/src/testApi.ts

```diff
@@ -187,11 +218,50 @@ export const DataRuntimeApi = {
 		sequence,
 		sequenceDeprecated,
 		agentScheduler,
+		tree: treeCurrent,
 	},
 };
 
 // #endregion
 
+/**
+ * Helper to load a package if the requested version is compatible.
+ * @param pkgEntry - The package entry to check and load.
+ * @param versionToInstall - The version of the package to install.
+ * @param modulePath - The path to the module.
+ * @returns The loaded package or undefined if not compatible.
+ */
+async function loadIfCompatible(
+	pkgEntry: PackageToInstall,
+	versionToInstall: string,
+	modulePath: string,
+): Promise<any> {
+	// Check if the requested version satisfies the minVersion requirement
+	if (semver.gte(versionToInstall, pkgEntry.minVersion)) {
+		return loadPackage(modulePath, pkgEntry.pkgName);
+	}
+	return undefined;
+}
+
+/**
+ * Helper to load multiple packages if their requested versions are compatible.
+ * @param packageEntries - The package entries to check and load.
+ * @param version - The version of the packages to install.
+ * @param modulePath - The path to the module.
+ * @returns An object containing the loaded packages.
+ */
+async function loadPackages(
+	packageEntries: PackageToInstall[],
+	version: string,
+	modulePath: string,
+) {
```

Comment:
It would be nice to add a return type here. That way we could also remove the `any` above